modules/local_contacts/modules.php

```diff
@@ -173,3 +238,31 @@ protected function output() {
             $this->trans('Cancel').'" /></div></form></div>';
     }
 }
+
+/**
+ * @subpackage import_local_contacts/output
+ */
+class Hm_Output_import_contacts_form extends Hm_Output_Module {
+    protected function output() {
+        $form_class = 'contact_form';
+        $button = '<input class="add_contact_submit" type="submit" name="import_contact" value="'.$this->trans('Add').'" />';
+        $notice = 'Please ensure your CSV header file follows the format: display_name, email_address, phone_number';
+        $title = $this->trans('Import from CSV file');
+
+        $sample_csv_data = "display_name,email_address,phone_number\nThomas Tester,test@example.org,1234567890";
```

Comment:
Please add this file as a static file directly in the git repo. No need to create it dynamically here.